lighthouse-core/audits/installable-manifest.js

```diff
@@ -97,6 +97,8 @@ const UIStrings = {
   'protocol-timeout': `Lighthouse could not determine if there was a service worker. Please try with a newer version of Chrome.`,
   /** Message logged when the web app has been uninstalled o desktop, signalling that the install banner state is being reset. */
   'pipeline-restarted': 'PWA has been uninstalled and installability checks resetting.',
+  /** Error message explaining that URL of the manifest uses a scheme that is not supported on Android. */
+  'scheme-not-supported-for-webapk': 'The manifest URL scheme is not supported on Android.',
```

Comment:
it would be kind of nice to have `` `The manifest URL scheme (${'urlScheme}') is not supported on Android.` `` though, and replace it with http/blob/data or whatever. The audit already uses the manifest url, so maybe not too annoying to include?

Reply:
I think we would need to handle the case where the manifest URL is `null`. It would probably be confusing if we use `null` as the URL scheme in the string, and I don't want to maintain two different versions.

We could suggest including the URL scheme as an argument on the installability error for `value0`.

Reply:
from my reading of `app_banner_manager_android.cc` that shouldn't be possible, so it seems ok to just fall back to `null` to keep the type checker happy?

> We could suggest including the URL scheme as an argument on the installability error for value0

I think the issue is how likely is that to happen?

Reply:
Sorry I was referring to our local var:
https://github.com/GoogleChrome/lighthouse/blob/586d4a2db6d6d98b8ca97c226651866b55728709/lighthouse-core/audits/installable-manifest.js#L200
The `WebAppManifest` will be null if there was no manifest data or a `PROTOCOL_TIMEOUT` occurred.